### PR TITLE
adjust to_html to bootstrap 5

### PR DESCRIPTION
**Pull request by achimgaedke**
Hi Oege!

Regarding conversion of static HTML output to bootstrap-5:

Most of it is done, there is one problem left: The tab function needs fixing and I don't understand how it works/worked. Do you have an idea?

I have removed some functions from `to_html`, as I discovered in the process that they are unused. This probably means that I actually don't need the `equal_height` parameter - which stretches cards to be aligned in height.

I am still a bit distracted by https://github.com/aio-libs/aiobotocore/pull/934 and #901 - let's see when I come back to this.

### CODE REVIEW

File: explainerdashboard/to_html.py [hunk 1]
```diff
@@ -129,47 +124,10 @@ def card(html:str, title:str=None, subtitle:str=None, border=True)->str:
...
-  <div class="card-footer">
-    <div class="w-100">
-    <a href="{url}" class="stretched-link">Go to dashboard</a>
-    </div>
-  </div>
-</div>
-"""
-
-
-def card_deck(*cards)->str:
```
Comment (oegedijk) [inline explainerdashboard/to_html.py:155]: this is no longer supported in bootstrap 5 I guess, so need to do something along these lines:

https://stackoverflow.com/questions/64311368/bootstrap-5-alpha-2-card-deck-example-not-showing-like-it-is-in-documentation

File: explainerdashboard/to_html.py [hunk 2]
```diff
@@ -281,14 +219,3 @@ def input(feature:str, value, disabled:bool=False)->str:
     <input id="{feature}" type="text" value="{value}" name="{feature}" {'disabled' if disabled else ''}>
 </div>
     """
-
-def jumbotron(title:str, description:str)->str:
```
Comment (oegedijk) [inline explainerdashboard/to_html.py:285]: against is used by ExplainerHub static html export, so need to create a replacement...

File: explainerdashboard/to_html.py [hunk 3]
```diff
@@ -6,16 +6,12 @@
     'rows',
     'fig',
     'card',
-    'dashboard_card',
-    'card_deck',
-    'card_rows',
```
[the remark this answers is not in the document]
Comment (achimgaedke) [reply explainerdashboard/to_html.py:11]: Hmmm, I think I made a mistake with VSCode's finding references... I will have a look again.

**Comment by achimgaedke** [comment]
Just pushed a new commit, there's still something wrong with the tabs. Will try again later.

File: explainerdashboard/to_html.py (hunk 1 above)
Comment (achimgaedke) [reply explainerdashboard/to_html.py:155]: I'll need to configure the card widths a bit more, most will need a simple `width=6` or 12 to fill half or the full space.

File: explainerdashboard/to_html.py (hunk 2 above)
Comment (achimgaedke) [reply explainerdashboard/to_html.py:285]: What is the preferred way to test the layout/s? Could you point me to a good test routine?

File: explainerdashboard/to_html.py [hunk 4]
```diff
@@ -226,42 +184,22 @@ def tabs(tabs_dict:dict)->str:
...
-            html += f"""<div class="tab-pane active" id="{tab_name}" role="tabpanel" aria-labelledby="{tab_name}-tab">\n {tab_contents} \n</div>\n"""
-        else:
-            html += f"""<div class="tab-pane" id="{tab_name}" role="tabpanel" aria-labelledby="{tab_name}-tab">\n {tab_contents} \n</div>\n"""
+        html += f"""<div class="tab-pane {'' if i else 'show active'}" id="{tab_name}"
+                    role="tabpanel" aria-labelledby="{tab_name}-tab">
+                    {tab_contents}
+                    </div>\n"""
 
     html += "\n</div>"
-
```
[the remark this answers is not in the document]
Comment (achimgaedke) [reply explainerdashboard/to_html.py:250]: Ok, reimplemented it wo jquery

File: explainerdashboard/to_html.py (hunk 1 above)
Comment (oegedijk) [reply explainerdashboard/to_html.py:155]: yeah, but idea is that this should be automatic and responsive. So you can't just hard code it at 6 or 12. Just not specifying the width of the cols should do the trick though I guess (bootstrap will divide the cols evenly).

File: explainerdashboard/to_html.py (hunk 2 above)
Comment (oegedijk) [reply explainerdashboard/to_html.py:285]: euhhh, not being a webdev, I think my routine when I originally developed it was

1. export to .html
2. open in browser
3. see if it looks good or if there are any errors.

(also need to include the static exports to the testing battery. In general working on updating the tests to pytest now, and need to extend the coverage)

**Comment by oegedijk** [comment]
made a few tweaks and now the card rows seem pretty nicely spaced

**Comment by oegedijk** [comment]
Give it a test run and see if you notice anything odd, otherwise I think it's ready to merge!

**Comment by achimgaedke** [comment]
Tried it out again and looks good to me.

**Comment by oegedijk** [comment]
cool, thanks again for all the work on this!

**Comment by oegedijk** [comment]
just released it as version 0.4.0

**Pull request merged**